### PR TITLE
Removes Sec, Peacekeeper and Clown Borgs

### PR DESCRIPTION
**Pull request by QuacksQ**
## About The Pull Request
Removes sec borgs and peace keeper borgs (Like god intended). 
Why remove them? Because they simply suck absolute ass to fight against and lack proper counters that normal carbon mobs like humans do have. You can hardly stun them and if one decides to chase you good luck because it has all access and can bolt doors with the press of a button.
## How Does This Help ***Gameplay***?
No more bullshit borgs that can chase you forever without stop. And no more weird peacekeeper borg.

## How Does This Help ***Roleplay***?
Minimal impact on roleplay.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary> 

image

</details>

## Changelog
:cl:
qol: Removed Sec borgs.
del: Removed Peacekeeper borgs.
del: Removed Clown borgs.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -184,10 +184,6 @@
 		"Janitor" = /obj/item/robot_model/janitor,
 		"Service" = /obj/item/robot_model/service,
 	)
-	if(!CONFIG_GET(flag/disable_peaceborg))
-		model_list["Peacekeeper"] = /obj/item/robot_model/peacekeeper
-	if(!CONFIG_GET(flag/disable_secborg))
```
Comment (RimiNosha) [inline code/modules/mob/living/silicon/robot/robot.dm:189]: Update config.txt please.

**Review by RimiNosha — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 1 above)
Comment (QuacksQ) [reply code/modules/mob/living/silicon/robot/robot.dm:189]: Config updated

**Review by RimiNosha — APPROVED** [review]
Finally, let's be rid of the shitty ruling nightmare borg, and the shitty meme "WEE WOO BAD LAW1 VIOLATOR" borg.

**Comment by QuacksQ** [comment]
> Finally, let's be rid of the shitty ruling nightmare borg, and the shitty meme "WEE WOO BAD LAW1 VIOLATOR" borg.

Remove all borgs next..?

**Comment by nevimer** [comment]
The roles are not the issues, the issues is that people have been accustomed to garbage gameplay loops and players in these roles. Security cyborgs are super easy to counter mechanically, and the removal of them is kinda sad to see, given that they work _pretty much everywhere else, where roleplay standards were above tgstation and skyrat's_. We shouldn't go right off the bat assuming our players will be awful based on previous assumptions from other servers.

You can literally print free stuns from a lathe as-is, just flash, ion gun, flashbang, laser pointer, the list goes on. Flashes are the easiest and most common tool to stun them, and it lasts forever, so anecdotes about them being hard to counter are only relevant for the nukie variants where they ARE flash immune. 

Now peacekeeper borgs only existed BECAUSE of human harm memes. Peacekeeper can go. Security borgs, last I checked, only have stun batons, disablers, and cuffs, and are quite literally useless at everything else. It's still a player chasing you if you're a criminal, carbon or not. The issue is if you remove all these things, you end up creating legitimate cases for anti-interaction mechanics such as printing off HOS ID's for more access, and security using jaws as standard carry. 

So far, as far as silicons go, nobody has thought outside the box for balance purposes. I propose putting this on hold for now.

**Comment by QuacksQ** [comment]
> Security cyborgs are super easy to counter mechanically,

I beg to differ tbh. Compared to a human all silicons, security borg or not lack a variety of counters. You cant do stamina damage to a borg, you cant slow a borg down through damage or stamina, you cant run into an area that a borg does not have access to because it has all access. All the items you listed are like the name suggests, Items. You have to print those and get them first if you want to deal with a borg, meanwhile with a human you shove them into a wall and they are down and crawling on the ground. 

I also strongly feel against having an AA secoff that isn't easy to counter.

**Comment by nevimer** [comment]
Security borgs are not the issue, knowing your enemy is important in battles. You might aswell remove the clownborg too due to infinite pies, which afaik do knowndown. You might not be able to damage a cyborg with your fists, but cyborgs are subject to the OLD 2014 era mechanics where haloss didn't exist so everything just added a sleep status effect. Which, if you play cyborgs a lot, is incredibly annoying because everyone has something to stun you at all times. Tools for a job kinda deal.

**Comment by QuacksQ** [comment]
> Security borgs are not the issue

Security borgs are the issue. 

An upgraded Sec borg with Ions, vtec and a T4 or T3 cell in our case is incredibly annoying to have to deal with. It can literally spam its disabler without consequence or a fear of running out any time soon. Meanwhile a normal sec off has to go recharge. The previous issue of no slowdown also still stands so I dont really see how knowing my enemy is gonna make it better.

**Comment by QuacksQ** [comment]
Thanks for reminding me about clown borgs tho

**Comment by RimiNosha** [comment]
I've already made it pretty clear on the discord that I'm for removing three types of borgs, plus AI due to complete garbage gameplay loops, poorly thought concepts, or S tier shitcode that has creeped it's way into just about everything (see uses for isrobot and isai).

**Comment by RimiNosha** [comment]
I have also made it abundantly clear that I am not okay with instant-win-based combat.

**Comment by nevimer** [comment]
Yeah, opens room for future replacements.

**Pull request merged**